src/com/luxoft/java/course/shevtsov/homework/algorithm/Main.java

```diff
@@ -1,21 +1,21 @@
 package com.luxoft.java.course.shevtsov.homework.algorithm;
 
-import java.util.Arrays;
+import com.luxoft.java.course.shevtsov.homework.algorithm.additional.NumberSymmetryCheck;
+import com.luxoft.java.course.shevtsov.homework.myUtilities.MyUtilities;
 
 public class Main {
 
     public static void main(String[] args) {
-        int[] ints = {-10, -8, -4, 0, 1, 5, 6, 9, 12, 20};
-        System.out.println("Array for searching:\t" + Arrays.toString(ints));
-        for (int searchNumber = -11; searchNumber <= 21; searchNumber++) {
-            System.out.println("-----------------------------------------------------------------");
-            System.out.println("search for " + searchNumber);
-            int resultIndex = BinarySearch.search(ints, searchNumber);
-
-            if (resultIndex == -1) {
-                System.out.println("There is NO such element.");
+        int number;
+        while (true) {
+            System.out.print("Input int number (should not exceed " + NumberSymmetryCheck.NUMBER_LENGTH
+                    + " characters). ");
+            System.out.print("For exit input '0': ");
```

Comment:
but zero it's valid number

Reply:
Agree. So how to do input method in loop right? Which value I have to input for exit from the loop? And which value should inputInt() method return in exit case?
One way is: I input “exit”-string. inputInt() generates manual exception (for example ExitExeption). This exception is processed in loop and gives command to exit. Is this a good solution?

Reply:
What about use enter numbers by one step? You no need request each number separately

Reply:
cool) done